R/sim1sbind.R

```diff
@@ -0,0 +1,58 @@
+#' Simulate nonlinear ligand-receptor binding with one binding site
+#'
+#' A sandbox to simulate and visualize random normal heteroscedastic response data. Variances enlarge with the value of y predicted by the model using a constant coefficeint of variation (cv). The data generating formula is derived from the one-site total binding model: y=Bmax*x/(x+kd). Failure errors in the plot fitting subfunction will occasionally happen due to the random data. These are more frequent with higher cv values. Just re-simulate with modified parameter values.
+#'
+#' @param x a vector of non-exponential linear scale values, usually representing dose or concentration, but can represent any stimulus.
```

Comment:
strike "but can represent any stimulus"  because this is binding, not stimuluation

Reply:
Done!